playbooks/gcp/openshift-cluster/publish_image.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Consider moving the logic under the role and have a shim that calls:

`import_role:` with `tasks_from: publish_image.yml`.